dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/group/MailDomainItem.java

```diff
@@ -0,0 +1,90 @@
+package edu.harvard.iq.dataverse.persistence.group;
+
+
+
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import java.util.Objects;
+
+@Entity
+public class MailDomainItem {
+
+    @Id
+    @GeneratedValue
+    private Long id;
+
+    @ManyToOne
+    private MailDomainGroup owner;
+
+    private String domain;
+
+    @Enumerated(EnumType.STRING)
+    private MailDomainProcessingType processingType;
+
+    // -------------------- CONSTRUCTORS --------------------
+
+    public MailDomainItem() { }
+
+    public MailDomainItem(String domain, MailDomainProcessingType processingType, MailDomainGroup owner) {
+        this.owner = owner;
+        this.domain = domain;
+        this.processingType = processingType;
+    }
+
+    // -------------------- GETTERS --------------------
+
+    public Long getId() {
+        return id;
+    }
+
+    public MailDomainGroup getOwner() {
+        return owner;
+    }
+
+    public String getDomain() {
```

Comment:
Maybe some comment describing what we store here? Especially info about the difference between domain with dot as first character vs no dot would be helpful